docs/reference/settings.md

```diff
@@ -198,6 +198,32 @@ package = false
 
 ---
 
+### [`sources`](#sources) {: #sources }
+
+The sources to use when resolving dependencies.
+
+`tool.uv.sources` enriches the dependency metadata with additional sources, incorporated
+during development. A dependency source can be a Git repository, a URL, a local path, or an
+alternative registry.
+
+See [Dependencies](https://docs.astral.sh/uv/concepts/dependencies/) for more.
```

Comment:
@zanieb -- I guess ideally this would be a relative path to the Markdown. But then it will _stay_ as a path in the JSON schema, which seems suboptimal. Wdyt?